src/main/java/com/elytradev/concrete/inventory/gui/widget/Widget.java

```diff
@@ -33,9 +33,8 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class WWidget {
+public class Widget {
```

Comment:
Yeah, I named it WWidget because WWidget is itself a widget. Just like Enum<E extends Enum<E>> *looks* crazy but informs you (and the compiler) about the class. People are going to be hunting down "all widgets" and this saves you time looking for strays by staying consistent.